### PR TITLE
remove default placeholder for subject description

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
partial fix for https://github.com/NeurodataWithoutBorders/nwb-guide/issues/229

The GUIDE inherits that aspect all the way from the NeuroConv base schema

### CODE REVIEW

File: src/neuroconv/schemas/base_metadata_schema.json [hunk 1]
```diff
@@ -143,7 +142,7 @@
             "U",
             "O"
           ],
-          "default": "U"
+          "default": ""
```
Comment (CodyCBakerPhD) [inline src/neuroconv/schemas/base_metadata_schema.json:145]: Also including a fix for the default placeholder for `sex`, but I'm not entirely sure if the form will complain about a `default` value that is not in the `enum`

@garrettmflynn

Comment (garrettmflynn) [reply src/neuroconv/schemas/base_metadata_schema.json:145]: At least for the form, this is equivalent to no default value. The final results object will have no "sex" key when an empty string is registered in the input.

Is there a reason you need a default value defined at all?

Comment (bendichter) [inline src/neuroconv/schemas/base_metadata_schema.json:145]:
```suggestion
```

Comment (CodyCBakerPhD) [reply src/neuroconv/schemas/base_metadata_schema.json:145]: No, removing it is the goal - I just wasn't sure what the forms expect

**Review by bendichter — APPROVED** [review]

**Pull request merged**